tests/PerformanceTest.php

```
@@ -0,0 +1,116 @@
+
+        // POST requests must not be cached
+        $_SERVER['REQUEST_METHOD'] = 'POST';
+        $this->assertFalse(PerformanceUtils::isCacheable());
+
+        // AJAX requests must not be cached
+        $_SERVER['REQUEST_METHOD'] = 'GET';
+        $_SERVER['HTTP_X_REQUESTED_WITH'] = 'xmlhttprequest';
+        $this->assertFalse(PerformanceUtils::isCacheable());
+
+        unset($_SERVER['HTTP_X_REQUESTED_WITH']);
+    }
```

PHPMD (2.15.0)

[error] testCachingEligibility accesses the super-global variable $_SERVER. (undefined)

(Superglobals)